src/react/account/AccountPoliciesList.tsx

```diff
@@ -0,0 +1,244 @@
+        label='Attach'
+        icon={<i className='fas fa-link'></i>}
+        onClick={() => history.push('attach-user-policy')}
+      />
+    </SpacedBox>);
+};
+
+const renderActionButtons = (rowValues, accountName) => {
+  const { arn, policyName } = rowValues;
+  const [accountPolicyName, path ] = policyName.split('_');
+  return (
+    <Box display='flex'>
+      <RenderAttachButton/>
+      <RenderEditButton policyName={accountPolicyName} path={path}/>
+      <CopyARNButton text={arn} />
+      <DeletePolicyAction policyName={accountPolicyName} path={path} arn={arn} accountName={accountName}/>
+    </Box>
+  );
+};
+
+const DeletePolicyAction = (rowValue: { policyName : string , path: string, arn: string, accountName: string}) => {
+  const { policyName, path, arn, accountName } = rowValue;
+  const IAMClient = useIAMClient();
+  const [showModal, setShowModal] = useState(false);
+  const pathCondition = path.includes('scality-internal');
+  const { status: listPoliciesStatus } = useQuery(getPoliciesQuery(policyName, notFalsyTypeGuard(IAMClient)));
+  const deletePolicyMutation = useMutation(
+    () => {
+      return notFalsyTypeGuard(IAMClient)
+        .deletePolicy(arn);
+    },
+    {
+      onSuccess: () => queryClient.invalidateQueries(getPoliciesQuery(accountName, notFalsyTypeGuard(IAMClient)).queryKey),
+    },
+  );
+
+  return (
+    <>
+      <DeleteConfirmation
+        show={showModal}
+        cancel={() => setShowModal(false)}
+        approve={() => {
+          deletePolicyMutation.mutate(policyName);
+        }}
+        titleText={`Permanently remove the following policy ${policyName} ?`}
+      />
+      <Box ml='0.6rem'>
+        <Button
+          style={{ height: spacing.sp24 }}
+          disabled
+          icon={<i className='fas fa-trash' />}
+          label=''
+          onClick={() => {
+            setShowModal(true);
+          }}
+          variant='danger'
+          tooltip={{ overlay: (listPoliciesStatus === 'loading' || pathCondition) ? 'Delete':'You cannot delete a predefined Scality Policy' }}
+        />
+      </Box>
+      {listPoliciesStatus === 'error' &&  <Banner
+        icon={<i className="fas fa-exclamation-triangle" />}
+        title="Error: Unable to delete policy"
+        variant="danger"
+      >
+        Error: Unable to delete policy.
+      </Banner>}
+    </>
+  );
+};
+
+const AccountPoliciesList = ({ accountName }: { accountName?: string }) => {
+  const history = useHistory();
+  const getQuery = (IAMClient: IAMClient) => getPoliciesQuery(notFalsyTypeGuard(accountName), IAMClient);
+  const getEntitiesFromResult = (data) => data.Policies;
+
+  const prepareData = (queryResult, search) => {
+    if (queryResult.firstPageStatus === 'success') {
+      const iamPolicies = queryResult && queryResult.data && queryResult.data.map((user) => {
+        return {
+          policyPath: user?.Path.substring(1),
+          policyName: user?.PolicyName + '_' + user?.Path,
```

Comment:
I am wondering wouldn't this be enough?
```suggestion
          policyName: user?.PolicyName,
```

Reply:
Yeah, sorry. I add it in order to address your previous comment: "We are missing a condition here. We only display this tooltip for Scality Internal Policies".
Do you have please any other solution in order to pass the path that we need in the condition (in order to enable/disable the tooltip) near the policyName ?